compiler-core/templates/docs-css/index.css

```diff
@@ -1229,3 +1230,9 @@ body.theme-dark .pride .blue {
     padding-top: 0;
   }
 }
+
+/* stabilizes computed-height of remote loaded images */
+main > h1:first-child + p {
```

Comment:
This is a very precise selector, what it it?

Reply:
Stabilizes computed-height of remote loaded images of the shield badges.
I agree this should be changed, but I wouldn't know where they get inserted.